tools/topology/topology2/cavs-sdw.conf

```diff
@@ -69,6 +69,11 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK       'mixout.1.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	DEEP_BUFFER_PIPELINE_ID_2	16
+	DEEP_BUFFER_PCM_ID_2		35
```

Comment:
@naveen-manohar can you remind me where this 35 comes from? I don't have access to my usual table of devices and this one does not ring a bell at all...

Reply:
Considered the next available PCM-ID similar to https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/cavs-rt5682.conf#L80

> arkham ~ # aplay -l
> **** List of PLAYBACK Hardware Devices ****
> card 0: sofsoundwire [sof-soundwire], device 0: Jack Out (*) []
>   Subdevices: 1/1
>   Subdevice #0: subdevice #0
> card 0: sofsoundwire [sof-soundwire], device 2: Speaker (*) []
>   Subdevices: 1/1
>   Subdevice #0: subdevice #0
> card 0: sofsoundwire [sof-soundwire], device 31: Deepbuffer Jack Out (*) []
>   Subdevices: 1/1
>   Subdevice #0: subdevice #0
> card 0: sofsoundwire [sof-soundwire], **device 35**: Deepbuffer Amps (*) []
>   Subdevices: 1/1
>   Subdevice #0: subdevice #0
> arkham ~ # aplay -Dhw:0,**35** -c 2 -r 48000 -d 10 -f S16_LE /root/pubg.wav

Reply:
give me a week to look into this, I don't recall us ever agreeing on all this.

Reply:
After checking my internal cheat sheet, I can confirm that the device 35 shows as reserved.

````
PCM28  RESERVED
PCM29  RESERVED
PCM30  playback	LowLatency
PCM31  playback	DeepBuffer (for low-power)
PCM32  playback	Media (e.g. movies)
PCM33  playback	Voice (Communication with a human)
PCM34  playback	Ultrasonics
PCM35  RESERVED
PCM36  RESERVED
PCM37  RESERVED
PCM38  RESERVED
PCM39  RESERVED
````

I don't mind reserving a Deep Buffer for amplifier endpoints, but note that we cannot keep adding in the PCM numbering both the content type and the endpoint. Are we e.g. going to have a LowLatency path for headphone AND speaker?

The "right" approach would be to keep the content-based stream generated by apps, and dynamically route to the current endpoint. What are the odds that we need a Deep buffer stream on the headphone and a Deep buffer stream on the amplifiers running concurrently?

That's what we had on the Merrifield and SKL drivers...The FE and BE were conditionally connected depending on a control switch.

@ranj063 @cujomalainey @ujfalusi @kv2019i FYI.

Reply:
++ @yongzhi1 fyi

Reply:
@plbossart jfyi, Chrome PnP team was asking deepbuf for Amp only, during their tests, they don't connect headset jack any more.
@sathya-nujella @cujomalainey can you help with below Query,
>> What are the odds that we need a Deep buffer stream on the headphone and a Deep buffer stream on the amplifiers running concurrently?

Reply:
> completely agree with you, thank you @plbossart @sathya-nujella we can close based on your comments.

If we have to choose one, then adding to speaker amp is needed as default BKM used by PnP team is that endpoint. Thanks!